### DISCUSSION TITLE
Stack Overflow Profile Icon

### DISCUSSION THREAD

**Discussion by charlieforward9** [community/community#48351] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Github and Stack Overflow have close ties in the developer experience, however, there is no icon for Stack Overflow when the social account link is added to the profile. 

<img width="176" alt="Screen Shot 2023-02-22 at 11 31 29 PM" src="https://user-images.githubusercontent.com/62311337/220821812-6ada8ec5-6c38-4c82-9fca-6d5d8774cd9f.png">
 Is this intentional or not thought of yet?
It would definitely be cool!

EDIT: Same with Spotify, listening to music when coding is my default method of coding!

**Comment by radar-blip** [comment]
Probably just not thought of yet. Doubt it was intentional.

**Reply by charlieforward9** [reply]
Are you on the team? If not, upvote to help make it seen!

**Reply by vedantmgoyal9** [reply]
@github-staff

**Comment by charlieforward9** [comment]
I realize this should be in the [profile discussion tab](https://github.com/orgs/community/discussions/categories/profile). Any easy way to transfer it?

**Reply by queenofcorgis** [reply]
Done ✅ Thanks!

**Comment by pjpscriv** [comment]
Same request raised in [this discussion](https://github.com/orgs/community/discussions/48265)

**Reply by charlieforward9** [reply]
Wow, what a coincidence that they were asked on the same day!

**Comment by pjpscriv** [comment]
For anyone interested - I created a little [userscript](https://gist.github.com/pjpscriv/6d29841107c93feeb042e730f29baea1) to do this

![image](https://github.com/user-attachments/assets/8803c6d6-e234-4a25-b4f5-a5b6e226b472)

**Reply by pjpscriv** [reply]
@charlieforward9 👀👀

![image](https://github.com/user-attachments/assets/72bf83a1-b46a-411f-938a-aafeb9a032d5)

**Reply by charlieforward9** [reply]
@pjpscriv How do I put this userscript into my profile?

**Reply by pjpscriv** [reply]
@charlieforward9 you add it to your browser with a [userscript manager](https://en.wikipedia.org/wiki/Userscript_manager?wprov=sfla1) - I use TamperMonkey.

So the steps would be:

1. Install the TamperMonkey extension to your browser - [here](https://chromewebstore.google.com/detail/tampermonkey/dhdgffkkebhmkfjojejmpbldmpobfkfo?hl=en) for Chrome, [here](https://addons.mozilla.org/en-CA/firefox/addon/tampermonkey/) for Firefox
2. Go to the gist: https://gist.github.com/pjpscriv/6d29841107c93feeb042e730f29baea1
3. Click the **Raw** button - iiuc TamperMonkey adds userscripts automatically from gists when you click this button
4. Go to your profile (or other profile with an S/O link - like mine 😁)
5. See the link updated ✨ _(very occasionally the formatting doesn't get applied, a page reload should fix this)_

**Reply by pjpscriv** [reply, marked as answer]
@charlieforward9 FYI if you're still interested in this, I converted the userscript into a Chrome extension: https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp - slightly less involved to set up now 😅
